numba/tests/npyufunc/test_vectorize_decor.py

```diff
@@ -129,12 +129,12 @@ def _test_target_unrecognized_arg(self, target, with_sig=True):
         b = np.array([3.0], dtype=np.float32)
         sig = [float32(float32, float32)]
         args = with_sig and [sig] or []
-        with self.assertRaises(NameError) as raises:
+        with self.assertRaises(KeyError) as raises:
             f = vectorize(*args, target=target, nonexistent=2)(vector_add)
             f(a, b)
         self.assertIn("Unrecognized options", str(raises.exception))
 
-class TestVectorizeUnrecognizedArg(BaseVectorizeUnrecognizedArg):
+class _(BaseVectorizeUnrecognizedArg):
```

Comment:
Is this name change intended?

Reply:
oh no

Reply:
fixed now https://github.com/numba/numba/pull/6814/commits/e5497c1fb0b61996c11de55f18ce72963ebefb58

Reply:
thanks